README.rst

```diff
@@ -121,6 +121,17 @@ NOTE: This configuration option currently only works for Linux.
     # Set notification expiry to 10 seconds
     export AUTO_NOTIFY_EXPIRE_TIME=10000
 
+    
+**Notification Urgency Level**
+
+By default non zero exit codes lead to critical notifications.
+On some systems (KDE) critical notifications will never expire.
```

Comment:
`On some systems (e.g. KDE) critical notifications are not desirable as they will never expire.`